docs/reference/services/document-vectorizer.md

```diff
@@ -0,0 +1,161 @@
+# document-vectorizer
+
+- [:material-account-group: Main author - HE-Arc](https://www.hes-so.ch/swiss-ai-center/equipe)
+- [:material-git: Code](https://github.com/swiss-ai-center/document-vectorizer-service)
+- [:material-kubernetes: Deployment configuration](https://github.com/swiss-ai-center/document-vectorizer-service/tree/main/kubernetes)
+- [:material-test-tube: Staging](https://document-vectorizer-swiss-ai-center.kube.isc.heia-fr.ch/docs)
+- [:material-factory: Production (not available yet)](https://document-vectorizer.swiss-ai-center.ch)
+
+This service uses langchain to vectorize a pdf document into a Facebook AI Similarity Search (FAISS)
+vectorstore.
+
+FAISS is a library that allows developers to quickly search for embeddings of multimedia documents
+that are similar to each other
+
+The API documentation is automatically generated by FastAPI using the OpenAPI
+standard. A user friendly interface provided by Swagger is available under the
+`/docs` route, where the endpoints of the service are described.
+
+This simple service only has one route `/compute` that takes a pdf document as
+input, and then returns a zip archive containing the vectorized document.
+
+This archive can be used in the
+[chatbot-ollama service](/reference/chatbot-ollama/). This chatbot
```

Comment:
You can use relative links (`../reference/chatbot-ollama.md`) instead of absolute links. This allows to `CTRL + Click` in our IDEs/GitHub Web UI to directly access the link(s). Material for MkDocs works great with this.